src/pages/background/services/signify.ts

```diff
@@ -47,6 +48,8 @@ const Signify = () => {
       _client = new SignifyClient(agentUrl, passcode, Tier.low, bootUrl);
       await _client.boot();
       await _client.connect();
+      const state = await getState();
+      await userService.setControllerId(state?.controller?.state?.i);
```

Comment:
@rodolfomiranda as client.state() is an async operation, it blocks the UI.
E.g if I try to fetch controllerId to attach with signin.
I have saved the controllerId into storage and fetch it from there.